mobile/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,72 @@
+import { type NavigationProp } from '@react-navigation/native';
+import React from 'react';
+
+import BackArrowImage from '#assets/img/icons/back-arrow.svg';
+import UserIcon from '#assets/img/icons/user.svg';
```

Comment:
Use Icon component

Reply:
@fshabanov could you please explain about Icon component. I dont have one for now, should i create it and add all SVGs there with purpose to use name property (create enum of existing SVGs and call required using Template literal) as you wrote here "Instead of iconSourceSvg pass name property". Or what should be inside icon component?

Reply:
@iamAlinaaa I quess, you can [see]( https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/114) in my pr what does @fshabanov means about Icon component.

Reply:
@OlenaYatseniuk thanks for sharing!
@iamAlinaaa that is exactly what I meant